ERROR: Service 'server' failed to build: The command '/bin/sh -c npm run build' returned a non-zero code: 1

Reproduced in v0.0.1-8-g4c22b1f

**NOTE**: This could have been avoided if #35 were fixed

### Steps to reproduce

Command

```bash
git describe --tag
docker-compose build --pull
```

Result

```text
gmacario@gmpowerhorse:~/github/gmacario/blobfishes (main)$ git describe --tag
v0.0.1-8-g4c22b1f
gmacario@gmpowerhorse:~/github/gmacario/blobfishes (main)$ docker-compose build --pull
Building static-pages
Step 1/7 : FROM node:14.17.3-alpine
14.17.3-alpine: Pulling from library/node
Digest: sha256:fb6cb918cc72869bd625940f42a7d8ae035c4e786d08187b94e8b91c6a534dfd
Status: Image is up to date for node:14.17.3-alpine
 ---> f5f48375fc5d
Step 2/7 : WORKDIR /usr/src/app
 ---> Using cache
 ---> 95f2d0208e36
Step 3/7 : COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
 ---> Using cache
 ---> f45e621402f1
Step 4/7 : RUN npm install --silent
 ---> Using cache
 ---> b3bc1a7951bd
Step 5/7 : COPY . .
 ---> Using cache
 ---> 3a972fda1004
Step 6/7 : RUN npm run build
 ---> Using cache
 ---> 350016c205bb
Step 7/7 : CMD ["ls", "-laR", "build/"]
 ---> Using cache
 ---> f788ed489797
Successfully built f788ed489797
Successfully tagged blobfishes_static-pages:latest
Building server
Step 1/7 : FROM node:14.17.3-alpine
14.17.3-alpine: Pulling from library/node
Digest: sha256:fb6cb918cc72869bd625940f42a7d8ae035c4e786d08187b94e8b91c6a534dfd
Status: Image is up to date for node:14.17.3-alpine
 ---> f5f48375fc5d
Step 2/7 : WORKDIR /usr/src/app
 ---> Using cache
 ---> 95f2d0208e36
Step 3/7 : COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
 ---> Using cache
 ---> 16f61c1356bb
Step 4/7 : RUN npm install --silent
 ---> Using cache
 ---> f63aeb5748d5
Step 5/7 : COPY . .
 ---> Using cache
 ---> 9014d4c7fa99
Step 6/7 : RUN npm run build
 ---> Running in 8bb5b6ce91da
npm ERR! missing script: build

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-22T01_25_17_865Z-debug.log
ERROR: Service 'server' failed to build: The command '/bin/sh -c npm run build' returned a non-zero code: 1
gmacario@gmpowerhorse:~/github/gmacario/blobfishes (main)$
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -24,7 +24,7 @@ COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
 # RUN ls -laR
 RUN npm install --silent
 COPY . .
-RUN npm run build
+RUN npm run build:server
 # RUN ls -la
 # RUN ls -laR dist/
 CMD ["node", "dist/app.js"]

